pkgs/applications/misc/please-cli/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, curl
+, fetchFromGitHub
+, jq
+, makeBinaryWrapper
+, please-cli
+, stdenv
+, testers
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "please";
+
+  pname = "please-cli";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "TNG";
+    repo = finalAttrs.pname;
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rJIR4eMhXL4K9iO7JxnkgWNsICV3hPQb0aobWNuHAG0=";
+  };
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $src/please.sh $out/bin/
+    makeBinaryWrapper $out/bin/please.sh $out/bin/${finalAttrs.name} \
+      --prefix PATH : ${lib.makeBinPath [ curl jq ]}
+  '';
+
+  passthru.tests = testers.testVersion {
+    package = please-cli;
+    command = "please --version";
+    version = "v${finalAttrs.version}";
+  };
+
+  meta = with lib; {
+    homepage = "https://github.com/TNG/please-cli";
+    description = "An AI helper script to create CLI commands based on GPT prompts";
+    platforms = platforms.all;
+    license = licenses.asl20;
+    maintainers = with lib.maintainers; [
```

Comment:
While there is no fixed consensus on the coding standard, I would use a single line for `maintainers` attribute, like it is done mostly everywhere else.